3회차/윤효근/4_창용마을무리의개수.py

```diff
@@ -1,3 +1,46 @@
 import sys
+from collections import defaultdict
+from collections import deque
```

Comment:
같은 라이브러리를 사용하는 경우 합쳐서 쓸 수 있습니다!
```suggestion
from collections import defaultdict, deque
```